edalize/apicula.py

```diff
@@ -0,0 +1,97 @@
+import os.path
+
+from edalize.edatool import Edatool
+from edalize.yosys import Yosys
+from importlib import import_module
+
+class Apicula(Edatool):
+
+    argtypes = ['vlogdefine', 'vlogparam']
+
+    @classmethod
+    def get_doc(cls, api_ver):
+        if api_ver == 0:
+            yosys_help = Yosys.get_doc(api_ver)
+            apicula_help = {
+                    'members' : [
+                        {'name' : 'pnr',
+                         'type' : 'String',
+                         'desc' : 'Select Place & Route tool. Legal values are *arachne* for Arachne-PNR or *next* for nextpnr. Default is arachne'}],
```

Comment:
drop the pnr option. I don't think arachne-pnr works for gowin devices so nextpnr is our only option here